gcloud/test_connection.py

```diff
@@ -374,7 +395,11 @@ def __init__(self, authorized=None):
 
     def authorize(self, http):
         self._called_with = http
-        return self._authorized
+
+        if callable(self._authorized):
```

Comment:
Yeah it's fine as-is. More than one way to skin a cat. This one's probably better after a second look.